config/olm/kubernetes/0.2.1/metadata/annotations.yaml

```diff
@@ -9,7 +9,3 @@ annotations:
   operators.operatorframework.io.metrics.mediatype.v1: metrics+v1
   operators.operatorframework.io.metrics.builder: operator-sdk-v1.5.0
```

Comment:
the latest operator sdk is 1.13.0, please double check that this value is correct